scripts/gulp/dev-server.js

```diff
@@ -32,118 +83,30 @@ function codeOfConductText() {
  * @constructor
  */
 function DevServer(port, config) {
+  const documentRoutes = buildDocumentRoutes();
+  const preformattedRoutes = {
+    '/document/code_of_conduct': './CODE_OF_CONDUCT',
+    '/document/license': './LICENSE',
+  };
```

Comment:
I don't think there is a whole lot of value in keeping these routes when we've got better content. I presume that was just to keep the existing landing page as-is?